include/switcher.php

```diff
@@ -251,6 +251,14 @@ public function the_languages( $links, $args = array() ) {
 			$walker = new PLL_Walker_List();
 		}
 
+		// Cast each elements to stdClass because $walker::walk() expects an array of objects.
+		$elements = array_map(
+			function( $element ) {
+				return (object) $element;
+			},
+			$elements
+		);
```

Comment:
Same with less code:
```suggestion
		foreach( $elements as &$element ) {
			$element = (object) $element;
		}
```

Reply:
Done in 8660249
Don't you mind I used the array index instead of passing by reference? I find it less confusing and more readable 😅